include/exadg/postprocessor/pointwise_output_generator_base.cpp

```diff
@@ -58,11 +58,11 @@ PointwiseOutputDataBase<dim>::print(dealii::ConditionalOStream & pcout) const
 template struct PointwiseOutputDataBase<2>;
 template struct PointwiseOutputDataBase<3>;
 
-template<int dim, typename VectorType>
+template<int dim, typename VectorType, typename Number>
```

Comment:
why do yo introduce another template parameter here?

Considering the instantiations below, it looks like `VectorType::value_type` and `Number` are always the same.